settings order.txt
item:
#    boiler                                        aa
#    steam engine                                  ab
    centrifuge                                    ac
    nuclear reactor                               ad
    heat boiler                                   ae
    steam turbine                                 af
#    solar panel                                   ag
#    accumulator                                   ah
#    substation                                    ai
#    lamp                                          aj
    assembler                                     ak
#    beacon                                        al
#    walkable beacon                               am
#    Logistics                                     ana (n: a-c)
#    Fluid handling                                anb (n: d-e)
#    inserter                                      ao (o: a-c)
    locomotive                                    ap
    wagon                                         aq
    fluid wagon                                   ar
#    repair pack                                   as

#    beacon productivity                           ba
#    Disable Productivity Limitation               bb
#    Behemoth Logistics Tweaks                     bc

#    inventory size                                ca
#    stack size                                    cb
#    enable stack size on Contruction Robot        cc
#    enable stack size on logistic Robot           cd
#    reach distance                                ce
#    mine distance                                 cf
#    minining speed                                cg
#    HP                                            ch
#    HP recovery                                   cia
#    Running speed                                 cib
#    bot speed                                     cj
#    no crafting time                              ck
#    Better Cable Visibility                       cl
#    wire shortcut                                 cm
#    wire crafting                                 cn
#DISABLED    alternativerecipe                     co
#DISABLED    cheatyrecipe                          cp
#    Assembler liquid Imput Output                 cq
#    Assembling Machine Imput Output               cr
#    Chemical Plant Imput Output                   cs
#    Oil Refineries Imput Output                   ct
#    mini equipment                                cu
#    ru-force-chemical-fuel                        cv
#    ru-force-equipment-categories                 cu

#    small electric pole wire reach                 da
#    small electric pole supply area                db
#    medium electric pole wire reach                dc
#    medium electric pole supply area               dd
#    big electric pole wire reach                   de
#    big electric pole supply area                  df
#    vanilla substation wire reach                  dg
#    vanilla substation supply area                 dh

#    Quick Start                                    ea
#    Quick Start Modular Armor                      eb
#    Quick Start Spidetron                          ac
#    Quick Start Train technology                   ed
#    Quick Start Robot technology                   ee

#    space exploration integration                  fa
#    ru-angel-integrations                          fb
#    ru-bob-integrations                            fc
#    ru-krastorio2-integrations                     fd
#    ru-LiquifyScienceandLab-integrations           fe

#    angel warehouse                                ga
#    angel logistic warehouse                       gb
#    krastorio2 medium container                    gc
#    krastorio2 medium logistic container           gd
#    krastorio2 big container                       ge
#    krastorio2 big logistic container              gf
#    Juicy Mods big generic chest                   gg
#    Juicy Mods big logistic chest                  gh
#    Juicy Mods big generic warehouse               gi
#    Juicy Mods big logistic warehouse              gj

#    angel pressuretank                             ha
#    krastorio2 big fluid storage                   hb
#    krastorio2 huge fluid storage                  hc
#    Juicy Mods fs 50                               hd
#    Juicy Mods fs 200                              he

changelog.txt
---------------------------------------------------------------------------------------------------
Version: 2.1.2
Date: xx/xx/2021
  Added:
    - added LiquifyScienceandLab integrations
  Changes:
---------------------------------------------------------------------------------------------------
Version: 2.1.1
Date: 10/02/2021
  Added:
    - added "ru-force-equipment-categories" for "spidertrontiers" with "bobvehicleequipment"
  Changes:
    - now "related_underground_belt" is indicated in the belt
---------------------------------------------------------------------------------------------------
Version: 2.1.0
Date: 26/01/2021
  Added:
    - settings to add fuel to various entity in the game
    - force chemical fuel
    - bob integration toggle
    - krastorio2 integration toggle
  Changes:
    - when disabling chemical input output even bob chemical plant will be disabled
    - when disabling oil refinery input output even bob oil refinery will be disabled
    - added compatibility with modmashsplinterairpurifier
    - Updated Wire Shortcut Script (If you need only this USE THE ORIGINAL!)
  Bugfixes:
    - omnimatter won't work with generalm input output enabled if the machine lever is higher than 5
    - technology image
    - basic/medium/big/behemoth filtered miniloader won't appear in game
  Notes:
    - if you disable the integration but enable another setting that work with that mod (not exclusive) the mod will be modified to, for example enabling force chemical fuel krastorio2 will be influenced even with krastorio2 integrations turned off
---------------------------------------------------------------------------------------------------
Version: 2.0.9
Date: 30/11/2020
  Added:
    - juicy mods storage size edit
    - juicy mods fluid storage size edit
    - added 8 extra miniloader with miniloader mod (they will disappear if you remove the miniloader mod) (more-miniloader original code)
    - deadlock miniloader and compacter with deadlock beltboxes loaders mod (they will disappear if you remove the miniloader mod) (more-miniloader original code)
  Changes:
    - Big logistics speed 150 > 180
    - Behemoth logistic speed 2000 > 360 (2 big belt)
    - behemoth logistic will allow speed change in settings 360 / 640 / 1280
    - changed maxium mine/reach distance 1000 > 100000
    - enabling logistics will no more change the vanilla express belt speed (60 > 45)
  Notes:
    - the belt speed was decreased due to factorio limitation
---------------------------------------------------------------------------------------------------
Version: 2.0.8
Date: 04/10/2020
  Added:
    - basic / medium / big / behemoth loader x1
    - mini equipment setting
    - infinite repair pack
  Changes:
    - basic / medium / big underground belt distance rebalance
  Bugfixes:
    - big / behemoth splitter west animation glitch
    - basic / medium / big beacon shadow 
---------------------------------------------------------------------------------------------------
Version: 2.0.7
Date: 27/10/2020
  Added:
    - basic / medium / big / behemoth loader x1
  Bugfixes:
    - crash with krastorio 2 installer and boiler enabled
    - crash with MicroFurnaceWipMod >= 1.0.7
---------------------------------------------------------------------------------------------------
Version: 2.0.6
Date: 26/10/2020
  Changes:
    - moved every item to RU category
    - changed boiler / steam engine ratio
    - changed boiler / steam engine consuption
    - changed boiler / steam engine effectivity
  Notes:
    - With space exploration postprocess installed effectivity on boiler and steam engine is disabled
---------------------------------------------------------------------------------------------------
Version: 2.0.5
Date: 23/10/2020
  Bugfixes:
    - some issues with krastorio 2 / aai-industry
---------------------------------------------------------------------------------------------------
Version: 2.0.4
Date: 22/10/2020
  Added:
    - added an option to remove productivity limitation from any module that has productivity in it
    - AAI industry boiler compatibility (toggle in the next update or a bit later)
    - custom tab to give a new order to the items (just testing all items will be moved in future)
  Changes:
    - rebalanced substation range
---------------------------------------------------------------------------------------------------
Version: 2.0.3
Date: 19/10/2020
  Added:
    - Depencies ZZZTweaks (obligatory)
    - Depencies ZZZCompatibility (optional)
    - Added character speed modifier
    - Added Junction Pole
  Changes:
    - rebalanced rubstation range
  Bugfixes:
    - steam engine craft not using proper items
    - transport belt upgrade planner
    - boiler upgrade planner
    - transport belt name
    - logistic technology name
    - boiler technology name
    - steam engine technology name
---------------------------------------------------------------------------------------------------
Version: 2.0.2
Date: 08/10/2020
  Bugfixes:
    - Fixed thumbnail
---------------------------------------------------------------------------------------------------
Version: 2.0.1
Date: 07/10/2020
  Bugfixes:
    - Fixed inserter compatibility with bobinserter
---------------------------------------------------------------------------------------------------
Version: 2.0.0
Date: 04/10/2020
  Added:
    - Basic / Medium / Big / Behemoth Transport Belt
    - Basic / Medium / Big / Behemoth Splitter
    - Basic / Medium / Big / Behemoth Underground Belt
    - Basic / Medium / Big / Behemoth Inserter
    - Basic / Medium / Big / Behemoth Stack Inserter
    - Basic / Medium / Big / Behemoth Boiler
    - Basic / Medium / Big / Behemoth Steam Engine
    - Beacon Productivity Setting (global)
    - Some angel warehouse settings
    - Some krastorio2 container settings
    - hp control
    - hp recovery control
    - Behemoth Quick Start 
    - Spidetron Quick Start

  Changes:

settings.lua
local selectable_armors = {"Nothing", "Modular armor", "Power armor", "Power armor MK2"}

-- Power Armor MK3
if mods["Power Armor MK3"] then
    table.insert(selectable_armors, "Power armor MK3")
    table.insert(selectable_armors, "Power armor MK4")
end

data:extend({
  --boiler OK
  {
    type = "string-setting",
    name = "ru-boiler",
    order = "aa",
    setting_type = "startup",
    default_value = "big",
    allowed_values =
    {
      "nothing",
      "basic",
      "medium",
      "big",
      "behemoth"
    },
  },
  --Steam Engine OK
  {
    type = "string-setting",
    name = "ru-steam-engine",
    order = "ab",
    setting_type = "startup",
    default_value = "big",
    allowed_values =
    {
      "nothing",
      "basic",
      "medium",
      "big",
      "behemoth"
    },
  },
  --Solar panel OK
  {
    type = "string-setting",
    name = "ru-solar-panel",
    order = "ag",
    setting_type = "startup",
    default_value = "big",
    allowed_values =
    {
      "nothing",
      "basic",
      "medium",
      "big",
      "behemoth"
    },
  },
  --accumulator OK
  {
    type = "string-setting",
    name = "ru-accumulator",
    order = "ah",
    setting_type = "startup",
    default_value = "big",
    allowed_values =
    {
      "nothing",
      "basic",
      "medium",
      "big",
      "behemoth"
    },
  },
  --substation OK
  {
    type = "string-setting",
    name = "ru-substation",
    order = "ai",
    setting_type = "startup",
    default_value = "big",
    allowed_values =
    {
      "nothing",
      "basic",
      "medium",
      "big",
      "behemoth"
    },
  },
  --lamp OK
  {
    type = "string-setting",
    name = "ru-lamp",
    order = "aj",
    setting_type = "startup",
    default_value = "big",
    allowed_values =
    {
      "nothing",
      "basic",
      "medium",
      "big",
--      "behemoth"
    },
  },
  --beacon OK
  {
    type = "string-setting",
    name = "ru-beacon",
    order = "al",
    setting_type = "startup",
    default_value = "big",
    allowed_values =
    {
      "nothing",
      "basic",
      "medium",
      "big",
--      "behemoth"
    },
  },
  --walkable beacon OK
  {
    type = "string-setting",
    name = "ru-walkable-beacon",
    order = "am",
    setting_type = "startup",
    default_value = "big",
    allowed_values =
    {
      "nothing",
      "basic",
      "medium",
      "big",
--      "behemoth"
    },
  },    
  --logistic = belt/splitter/underground OK
  {
    type = "string-setting",
    name = "ru-Logistic",
    order = "ana",
    setting_type = "startup",
    default_value = "big",
    allowed_values =
    {
      "nothing",
      "basic",
      "medium",
      "big",
      "behemoth"
    },
  },
  --fluid handling = pump/pipe/pipe to ground/storage tank OK
  {
    type = "string-setting",
    name = "ru-fluid-handling",
    order = "anb",
    setting_type = "startup",
    default_value = "big",
    allowed_values =
    {
      "nothing",
      "basic",
      "medium",
      "big",
      "behemoth"
    },
  },
  --inserter OK
  {
  type = "string-setting",
  name = "ru-inserter",
  order = "ao",
  setting_type = "startup",
  default_value = "big",
  allowed_values =
    {
      "nothing",
      "basic",
      "medium",
      "big",
      "behemoth"
    },
  },
  --Infinite Repairer OK
  {
    type = "bool-setting",
    name = "ru-Infinite-Repairer",
    order = "as",
    setting_type = "startup",
    default_value = "true",
  },

  --beacon Productivity
  {
    type = "bool-setting",
    name = "ru-beacon-Productivity",
    order = "ba",
    setting_type = "startup",
    default_value = "true",
  },
  --beacon Productivity
  {
    type = "bool-setting",
    name = "ru-Disable-Productivity-Limitation",
    order = "bb",
    setting_type = "startup",
    default_value = "true",
  },
  --behemoth logistics tweaks
  {
    type = "string-setting",
    name = "ru-behemoth-logistics-tweaks",
    order = "bc",
    setting_type = "startup",
    default_value = "360",
    allowed_values =
    {
      "360",
      "640",
      "1280",
    },
  },


  --Inventory size OK
  {
    type = "int-setting",
    name = "ru-inventory-size",
    order = "ca",
    setting_type = "startup",
    default_value = 80,
    minimum_value = 20,
    maximum_value = 2000,
  },
  --stack size multiplier OK
  {
    type = "double-setting",
    name = "ru-stack-Size",
    order = "cb",
    setting_type = "startup",
    default_value = 10,
    minimum_value = 1,
    maximum_value = 2147483647,
  },
  --enable stack size Contruction Robot OK
  {
    type = "bool-setting",
    name = "ru-stack-Size-CRobot",
    order = "cc",
    setting_type = "startup",
    default_value = "true",
  },
  --enable stack size Logistic Robot OK
  {
    type = "bool-setting",
    name = "ru-stack-Size-LRobot",
    order = "cd",
    setting_type = "startup",
    default_value = "true",
  },
  --reach distance OK
  {
    type = "int-setting",
    name = "ru-reach-distance",
    order = "ce",
    setting_type = "startup",
    default_value = 6,
    minimum_value = 1,
    maximum_value = 100000,
  },
  --Mining reach OK
  {
    type = "int-setting",
    name = "ru-mine-reach",
    order = "cf",
    setting_type = "startup",
    default_value = 6,
    minimum_value = 1,
    maximum_value = 100000,
  },
  --Mining speed OK
  {
    type = "double-setting",
    name = "ru-mining-speed",
    order = "cg",
    setting_type = "startup",
    default_value = 1,
    minimum_value = 0.01,
    maximum_value = 1000,
  },
  --hp OK
  {
    type = "int-setting",
    name = "ru-hp",
    order = "ch",
    setting_type = "startup",
    default_value = 250,
    minimum_value = 1,
    maximum_value = 250000,
  },
  --hp recovery OK
  {
    type = "double-setting",
    name = "ru-hp-recovery",
    order = "cia",
    setting_type = "startup",
    default_value = 0.15,
    minimum_value = 0.1,
    maximum_value = 1500,
  },
  --running speed
  {
    type = "double-setting",
    name = "ru-running-speed",
    order = "cib",
    setting_type = "startup",
    default_value = 0,
    minimum_value = -100,
    maximum_value = 100,
  },
  --bot speed OK
  {
    type = "double-setting",
    name = "ru-botspeed",
    order = "cj",
    setting_type = "startup",
    default_value = 0,
    minimum_value = 0,
    maximum_value = 1,
  },
  --No Crafting Time OK
  {
    type = "bool-setting",
    name = "ru-No-Crafting-Time",
    order = "ck",
    setting_type = "startup",
    default_value = "false",
  },
  --RU Better Cable Visibility OK
  {
    type = "bool-setting",
    name = "ru-Better-Cable-Visibility",
    order = "cl",
    setting_type = "startup",
    default_value = "true",
  },
  --Enable wire Shortcuts OK
  {
    type = "bool-setting",
    name = "ru-Enable-Wire-Shortcuts",
    order = "cm",
    setting_type = "startup",
    default_value = "true",
  },
  --mantain Wire crafting OK
  {
    type = "bool-setting",
    name = "ru-wire-shortcuts-is-retain-wire-crafting",
    order = "cna",
    setting_type = "startup",
    default_value = false,
  },
  --mantain Wire cutter OK
  {
    type = "bool-setting",
    name = "ru-wire-shortcuts-is-advanced-cutter",
    order = "cnb",
    setting_type = "runtime-per-user",
    default_value = false,
  },
  --alternativerecipe DISABLED NO
  {
    type = "bool-setting",
    name = "ru-alternativerecipe",
    hidden = "true",
    order = "co",
    setting_type = "startup",
    default_value = "false",
  },
  --cheatyrecipe DISABLED NO
  {
    type = "bool-setting",
    name = "ru-cheatyrecipe",
    hidden = "true",
    order = "cp",
    setting_type = "startup",
    default_value = "false",
  },
  --ru-Machine-liquid-Imput-Output OK
  {
    type = "bool-setting",
    name = "ru-Machine-liquid-Imput-Output",
    order = "cq",
    setting_type = "startup",
    default_value = "false",
  },
  --ru-Assembling-Machine-Imput-Output OK
  {
    type = "bool-setting",
    name = "ru-Assembling-Machine-Imput-Output",
    order = "cr",
    setting_type = "startup",
    default_value = "true",
  },
  --ru-Chemical-Plant-Imput-Output OK
  {
    type = "bool-setting",
    name = "ru-Chemical-Plant-Imput-Output",
    order = "cs",
    setting_type = "startup",
    default_value = "true",
  },
  --ru-Oil-Refineries-Imput-Output OK
  {
    type = "bool-setting",
    name = "ru-Oil-Refineries-Imput-Output",
    order = "ct",
    setting_type = "startup",
    default_value = "true",
  },
  --ru-MiniEquipment OK
  {
    type = "bool-setting",
    name = "ru-MiniEquipment",
    order = "cu",
    setting_type = "startup",
    default_value = "true",
  },
  --ru-force-chemical-fuel
  {
  type = "bool-setting",
  name = "ru-force-chemical-fuel",
  order = "cv",
  setting_type = "startup",
  default_value = "true",
  },
  --ru-force-equipment-categories
  {
    type = "bool-setting",
    name = "ru-force-equipment-categories",
    order = "cu",
    setting_type = "startup",
    default_value = "true",
  },

  --small electric pole wire reach OK
  {
    type = "double-setting",
    name = "small-electric-pole-max-wire-distance",
    order = "da",
    setting_type = "startup",
    maximum_value = 64,
    default_value = 7.5,
    minimum_value = 2.5,
  },
  --small electric pole supply area OK
  {
    type = "double-setting",
    name = "small-electric-pole-supply-area-distance",
    order = "db",
    setting_type = "startup",
    maximum_value = 128,
    default_value = 5,
    minimum_value = 3,
  },
  --medium electric pole wire reach OK
  {
    type = "double-setting",
    name = "medium-electric-pole-max-wire-distance",
    order = "dc",
    setting_type = "startup",
    maximum_value = 64,
    default_value = 9,
    minimum_value = 2.5,
  },
  --medium electric pole supply area OK
  {
    type = "double-setting",
    name = "medium-electric-pole-supply-area-distance",
    order = "dd",
    setting_type = "startup",
    maximum_value = 128,
    default_value = 7,
    minimum_value = 3,
  },
  --big electric pole wire reach OK
  {
    type = "double-setting",
    name = "big-electric-pole-max-wire-distance",
    order = "de",
    setting_type = "startup",
    maximum_value = 64,
    default_value = 30,
    minimum_value = 3,
  },
  --big electric pole supply area OK
  {
    type = "double-setting",
    name = "big-electric-pole-supply-area-distance",
    order = "df",
    setting_type = "startup",
    maximum_value = 128,
    default_value = 4,
    minimum_value = 4,
  },
  --vanilla substation wire reach OK
  {
    type = "double-setting",
    name = "substation-max-wire-distance",
    order = "dg",
    setting_type = "startup",
    maximum_value = 64,
    default_value = 18,
    minimum_value = 3,
  },
  --vanilla substation supply area OK
  {
    type = "double-setting",
    name = "substation-supply-area-distance",
    order = "dh",
    setting_type = "startup",
    maximum_value = 128,
    default_value = 18,
    minimum_value = 4,
  },

  --Quick Start OK
  {
    type = "string-setting",
    name = "ru-quick-start",
    order = "ea",
    setting_type = "startup",
    default_value = "Nothing",
    allowed_values = 
    {
      "Nothing", 
      "Basic", 
      "Medium", 
      "Big", 
      "Huge",
      "Behemoth"
    },
  },
  --Quick Start Armor OK
  {
    type = "string-setting",
    name = "ru-armor-quick-start",
    order = "eb",
    setting_type = "startup",
    default_value = "Nothing",
    allowed_values = selectable_armors,
  },
  --Quick Start spidertron OK
  {
    type = "bool-setting",
    name = "ru-quick-start-spidetron",
    order = "ec",
    setting_type = "startup",
    default_value = "false",
  },
  --Quick Start Train technology OK
  {
    type = "bool-setting",
    name = "ru-train-technology",
    order = "ed",
    setting_type = "startup",
    default_value = "false",
  },
  --Quick Start Robot technology OK
  {
    type = "bool-setting",
    name = "ru-robot-technology",
    order = "ee",
    setting_type = "startup",
    default_value = "false",
  },

  --space exploration integration OK
  {
    type = "bool-setting",
    name = "ru-space-exploration-integrations",
    hidden = "true",
    order = "fa",
    setting_type = "startup",
    default_value = "true",
  },
  --angel integration OK
  {
    type = "bool-setting",
    name = "ru-angel-integrations",
    hidden = "true",
    order = "fb",
    setting_type = "startup",
    default_value = "true",
  },
  --bob integration OK
  {
    type = "bool-setting",
    name = "ru-bob-integrations",
    hidden = "true",
    order = "fc",
    setting_type = "startup",
    default_value = "true",
  },
  --krastorio2 integration OK
  {
    type = "bool-setting",
    name = "ru-krastorio2-integrations",
    hidden = "true",
    order = "fd",
    setting_type = "startup",
    default_value = "true",
  },
  --LiquifyScienceandLab integration OK
  {
    type = "bool-setting",
    name = "ru-LiquifyScienceandLab-integrations",
    hidden = "true",
    order = "fe",
    setting_type = "startup",
    default_value = "true",
  },

  --angel warehouse OK
  {
    type = "int-setting",
    name = "ru-angels-warehouse",
    hidden = "true",
    order = "ga",
    setting_type = "startup",
    default_value = 1800,
    minimum_value = 1,
    maximum_value = 65535,
  },
  --angel logistic warehouse OK
  {
    type = "int-setting",
    name = "ru-angels-logistic-warehouse",
    hidden = "true",
    order = "gb",
    setting_type = "startup",
    default_value = 1800,
    minimum_value = 1,
    maximum_value = 65535,
  },
  --krastorio2 medium container OK
  {
    type = "int-setting",
    name = "ru-Krastorio2-medium-container",
    hidden = "true",
    order = "gc",
    setting_type = "startup",
    default_value = 400,
    minimum_value = 1,
    maximum_value = 10000,
  },
  --krastorio2 medium logistic container OK
  {
    type = "int-setting",
    name = "ru-Krastorio2-medium-logistic-container",
    hidden = "true",
    order = "gd",
    setting_type = "startup",
    default_value = 400,
    minimum_value = 1,
    maximum_value = 10000,
  },
  --krastorio2 big container OK
  {
    type = "int-setting",
    name = "ru-Krastorio2-big-container",
    hidden = "true",
    order = "ge",
    setting_type = "startup",
    default_value = 1800,
    minimum_value = 1,
    maximum_value = 65535,
  },
  --krastorio2 big logistic container OK
  {
    type = "int-setting",
    name = "ru-Krastorio2-big-logistic-container",
    hidden = "true",
    order = "gf",
    setting_type = "startup",
    default_value = 1800,
    minimum_value = 1,
    maximum_value = 65535,
  },
  --Juicy Mods big generic chest OK
  {
    type = "int-setting",
    name = "ru-juicy-mods-big-generic-chest",
    hidden = "true",
    order = "gg",
    setting_type = "startup",
    default_value = 400,
    minimum_value = 1,
    maximum_value = 10000,
  },
  --Juicy Mods big logistic chest OK
  {
    type = "int-setting",
    name = "ru-juicy-mods-big-logistic-chest",
    hidden = "true",
    order = "gh",
    setting_type = "startup",
    default_value = 400,
    minimum_value = 1,
    maximum_value = 10000,
  },
  --Juicy Mods generic warehoude OK
  {
    type = "int-setting",
    name = "ru-juicy-mods-generic-warehoude",
    hidden = "true",
    order = "gi",
    setting_type = "startup",
    default_value = 2000,
    minimum_value = 1,
    maximum_value = 65535,
  },
  --Juicy Mods logistic warehoude OK
  {
    type = "int-setting",
    name = "ru-juicy-mods-logistic-warehoude",
    hidden = "true",
    order = "gj",
    setting_type = "startup",
    default_value = 2000,
    minimum_value = 1,
    maximum_value = 65535,
  },

  --angel pressuretank OK
  {
    type = "int-setting",
    name = "ru-Angel-Pressuretank",
    hidden = "true",
    order = "ha",
    setting_type = "startup",
    default_value = 100000,
    minimum_value = 1,
    maximum_value = 100000000,
  },
  --Krastorio2 big fluid storage OK
  {
    type = "int-setting",
    name = "ru-kr-fluid-storage-1",
    hidden = "true",
    order = "hb",
    setting_type = "startup",
    default_value = 1000,
    minimum_value = 1,
    maximum_value = 100000000,
  },
  --Krastorio2 huge fluid storage OK
  {
    type = "int-setting",
    name = "ru-kr-fluid-storage-2",
    hidden = "true",
    order = "hc",
    setting_type = "startup",
    default_value = 100000,
    minimum_value = 1,
    maximum_value = 100000000,
  },
  --Juicy Mods fs 50 OK
  {
    type = "int-setting",
    name = "ru-juicy-mods-fs-50",
    hidden = "true",
    order = "hd",
    setting_type = "startup",
    default_value = 1000,
    minimum_value = 1,
    maximum_value = 100000000,
  },
  --Juicy Mods fs 200 OK
  {
    type = "int-setting",
    name = "ru-juicy-mods-fs-200",
    hidden = "true",
    order = "he",
    setting_type = "startup",
    default_value = 100000,
    minimum_value = 1,
    maximum_value = 100000000,
  },
})